pages/api/tool/create.ts

```diff
@@ -0,0 +1,13 @@
+import { Tool } from "database/models/tool";
```

Comment:
The usual naming convention for the post file is `/api/tool/post` (at least this is what is being used elsewhere in the codebase)